### PR TITLE
Update docs

### LINKED ISSUE #226: Docs: Example links on blob/master/docs/index.md appear to be broken

### PR DESCRIPTION
**Pull request by mlabieniec**
- update docs index to include examples, Fixes #226

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -72,3 +72,88 @@ Now run your application as normal:
 ```bash
 react-native run-ios
 ```
+
+## Examples
+
+AWS Amplify supports many category scenarios such as Auth, Analytics, APIs and Storage as outlined in the [Developer Guide](docs/media/developer_guide.md). A couple of samples are below.
```
Comment (elorzafe) [inline docs/index.md]: The link is broken, should remove docs/

File: docs/index.md [hunk 2]
```diff
@@ -72,3 +72,88 @@ Now run your application as normal:
...
+
+### 1. Collect user session metrics
+
+By default, AWS Amplify can send user session information as metrics with a few lines of code:
+
+```js
+import Amplify, { Analytics } from 'aws-amplify';
+import aws_exports from './aws-exports';
+
+Amplify.configure(aws_exports);
```
Comment (elorzafe) [inline docs/index.md]: Maybe add where to get aws_exports file? Like on the other docs.

**Comment by mlabieniec** [comment]
@elorzafe updated

**Comment by mlabieniec** [comment]
Updated to include Fix for #229

**Comment by powerful23** [comment]
also fixed issue #242

**Pull request merged**